src/test/java/com/techsophy/tsf/account/controller/UserFormDataControllerTest.java

```diff
@@ -77,11 +83,17 @@ void getUserDetailsOfLoggedInUserNotFound() throws JsonProcessingException {
     @CsvSource({"1,1", ",1"}) //here i am verifying with passing id and their invocation for service layer
     void updateUserDetailsOfLoggedInUserSuccessWithId(String args,int invocation) throws JsonProcessingException {
         Map<String,Object> map = new HashMap<>();
-        map.put("abc","abc");
+        map.put("userName","nandini");
+        map.put("emailId","nandini.k@techsophy.com");
+        map.put("groups","awgment");
+        map.put("roles","awgment-account-all");;
         UserFormDataSchema userFormDataSchema = new UserFormDataSchema(map,args,"abc");
         List<Map<String,Object>> list = new ArrayList<>();
         map.put("id","1");
         list.add(map);
+        AuditableData auditableData = new AuditableData("abc", Instant.now(),"abc",Instant.now());
+        Mockito.when(userFormDataService.getUserFormDataByUserId("1",false)).thenReturn(auditableData);
+        Mockito.when(objectMapper.convertValue(any(), ArgumentMatchers.eq(UserFormDataSchema.class))).thenReturn(userFormDataSchema);
```

Comment:
add a check/test case to ensure that the username/role/group are not modified by api call